src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -1030,6 +1041,25 @@ public void checkOptionsValid() {
               + " specified a class literal file and --use-class-literals=NONE");
     }
 
+    if (literal_tfidf && literals_level == ClassLiteralsMode.NONE) {
+      throw new RandoopUsageError(
+          "Invalid parameter combination:"
+              + " specified --literal-tfidf and --use-class-literals=NONE");
+    }
+
+    if (literal_tfidf && literals_file.contains("CLASSES")) {
+      throw new RandoopUsageError(
+          "Invalid parameter combination: "
+              + "cannot use both --literal-tfidf and --literals-file=CLASSES. "
```

Comment:
Is there anything wrong with this?  `--literal-tfidf` implies `--literals-file=CLASSES`, so the latter is redundant but not contradictory.